design/fast_finality.md

```diff
@@ -13,10 +13,12 @@ described in the [Bootstrapping Strategy design doc](./Bootstrapping_strategy.md
 ## Requirements
 
 * Finality
-  * Transaction *soft* finality (finality guaranteed by the sequencer) is achieved in under one second
+  * Transaction *soft* finality (finality guaranteed by the sequencer) has a median duration of one second, from when 
```

Comment:
I think the 1 second is a round trip to the user.
T0: User submits tx
T1: User receives tx receipt and events

where median (T1-T0)=1s